tools/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.csproj

```diff
@@ -29,11 +29,12 @@
       <IncludeBuildOutput>false</IncludeBuildOutput>
       <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
       <NoWarn>NU5100</NoWarn>
+      <ProtobufVersion>26.0</ProtobufVersion>
```

Comment:
We should update to 26.1, released earlier today. In a follow-up PR?

Reply:
Updated